docs/tutorials/pytorch-mnist-local-qs.rst

```diff
@@ -0,0 +1,114 @@
+.. _pytorch_mnist_quickstart:
+
+#########################################
+ Run Your First Experiment in Determined
+#########################################
+
+.. meta::
+   :description: Learn how to integrate the PyTorch MNIST model into Determined AI using only a single CPU or GPU.
+   :keywords: PyTorch API,MNIST,model developer,quickstart
+
+In this tutorial, we’ll show you how to integrate a training example with the Determined
+environment. We'll run our experiment on a local training environment requiring only a single CPU or
+GPU.
+
+.. note::
+
+   This tutorial is recommended as an introduction for model developers who are new to Determined
+   AI.
+
+**Objective**
+
+Our goal is to integrate the `PyTorch MNIST training example
+<https://github.com/pytorch/examples/blob/main/mnist/main.py>`_ into Determined in four steps:
+
+-  Download and extract the files
+-  Set up our training environment
+-  Run the experiment
+-  View the experiment in our browser
+
+**Prerequisites**
```

Comment:
Do we want to include pip-installing determined?

Reply:
done